packages/ui/components/button/Button.tsx

```diff
@@ -48,9 +48,9 @@ export const buttonClasses = cva(
       },
       color: {
         primary:
-          "bg-brand-default hover:bg-brand-emphasis focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-offset focus-visible:ring-brand-default text-brand disabled:bg-brand-subtle disabled:text-brand-subtle disabled:opacity-40 disabled:hover:bg-brand-subtle disabled:hover:text-brand-default disabled:hover:opacity-40",
+          "bg-brand-default hover:bg-brand-emphasis focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-offset focus-visible:ring-brand-default text-brand disabled:bg-brand-subtle disabled:text-brand-subtle disabled:opacity-40 disabled:hover:bg-brand-subtle disabled:hover:text-brand-default disabled:hover:opacity-40 dark:bg-white dark:text-black",
         secondary:
```

Comment:
These values should be correct already - without the dark: prefix

If they were not working in a specific area of the app (onboarding) the variables may not have been set